sycl/doc/extensions/README.md

```diff
@@ -1,3 +1,34 @@
 # Extensions
 
-This is where documents can be found that propose extensions to the SYCL specification.
+The directory contains documents that describe DPC++ extensions to SYCL
+specification.
+
+DPC++ extensions status:
+
```

Comment:
`printf` extension seems to be missing.

Reply:
@bader , do we have an extension doc for the printf extension?
@smaslov-intel  I haven't added non-Intel extensions intentionally.
@pvchupin Should we track non-Intel extensions here? What if they are not implemented so far?

Reply:
> @bader , do we have an extension doc for the printf extension?

I've requested it [here](https://github.com/intel/llvm/issues/1154#issuecomment-589566270), but there were updates yet. @AlexeySachkov any updates?

Reply:
@pvchupin Should we track non-Intel extensions here? What if they are not implemented so far?

Reply:
I think this table purpose is to clarify what we support planning to prototype in the nearest future (Intel or non-Intel doesn't matter).
In particular we have specialization constants implementation, so we should add it. We should probably add generalization too. I don't think we should extend this list to all existing drafts, only to those we are considering or prototyped already.

Reply:
Ok, are you OK if these extensions are added to the table in a separate PR(preferably by the authors)?